ios/NedaaWidget/PrayerCountdownView.swift

```diff
@@ -0,0 +1,168 @@
+import WidgetKit
+import SwiftUI
+import Intents
+
+struct CountdownViewProvider: IntentTimelineProvider {
+    typealias Entry = PrayerEntry
+    typealias Intent = ConfigurationIntent
+    
+    func placeholder(in context: Context) -> PrayerEntry {
+        PrayerEntry(date: Date(), configuration: ConfigurationIntent(), nextPrayer: PrayerData(name:NSLocalizedString("fajr", comment: "fajr"), date: Date().addingTimeInterval(TimeInterval(60 * 5))), previousPrayer: PrayerData(name: NSLocalizedString("isha", comment: "isha"), date: Date().addingTimeInterval(3600)) )
```

Comment:
Weird using `TimeInterval(60*5)` and `3600` seems inconsistent

Reply:
Probably I was testing to see the difference between placeholder and snapshot but it doesn't really matter I think will only show when we select the widget as a preview